actions/maven-dependency/main.go

```diff
@@ -44,6 +45,13 @@ func main() {
 		panic(fmt.Errorf("artifact_id must be specified"))
 	}
 
+	versionRegex, ok := inputs["version_regex"]
+	if !ok {
+		fmt.Println(`No version_regex set, using default: ^[\d]+\.[\d]+\.[\d]+/$`)
+		versionRegex = `^[\d]+\.[\d]+\.[\d]+$`
```

Comment:
The original version pattern did not have a leading ^ symbol, I presume this does not matter?